README.md

```diff
@@ -101,11 +101,7 @@ Licensed under either of
 
 at your option.
 
-Some files used for tests are under different licenses:
-
-- The font file `Arimo-VariableFont_wght.ttf` in `/parley/tests/assets/arimo_fonts/` is licensed solely as documented in that folder (and is licensed under the Apache License, Version 2.0).
-- The font file `Roboto-Regular.ttf` in `/parley/tests/assets/roboto_fonts/` is licensed solely as documented in that folder (and is licensed under the Apache License, Version 2.0).
-- The font file `NotoKufiArabic-Regular.otf` in `/parley/tests/assets/noto_fonts/` is licensed solely as documented in that folder (and is licensed under the SIL Open Font License, Version 1.1).
+Some files used for tests and benchmarks are under different licenses. See `./parley_dev/README.md` for details.
```

Comment:
This still needs to be reflected in `./parley/README.md` (i.e. the "Some files used for tests are under different licenses:" section can just be removed).

Reply:
🤦 - Thanks for this catch. Removed in https://github.com/linebender/parley/pull/405/commits/2e61fa686eca25db940883ff6058bfd289429a9e